### PR TITLE
Upload .tgz file for Linux releases.

### PR DESCRIPTION
**Pull request by kasperl**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -30,14 +30,18 @@ jobs:
             build/linux/
             build/macos/
             build/windows/
-      - name: Upload jag linux
-        if: github.event_name == 'release'
+      - name: Create archive
+        working-directory: ./build/linux
+        run: |
+          tar cvzf jag.tgz jag
```
Comment (jstroem) [inline .github/workflows/build.yml:36]: `-cvzf`

**Review by jstroem — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (kasperl) [reply .github/workflows/build.yml]: Done.

**Comment by floitsch** [comment]
Could we either make this a `gz` (no 'tar') or move the executable into a folder.
It's generally expected that a tar.gz extracts into a folder of the same name.

**Comment by kasperl** [comment]
I think we need this as a tar-ball, because we want to preserve file permissions (u+x). I get the desire to not pollute the working directory with lots of stuff when you untar, but since this is a single file, it feels a bit odd to prefer unpacking that into a single directory with a single file within.

**Pull request merged**